suse_migration_services/units/setup_host_network.py

```diff
@@ -62,11 +66,19 @@ def main():
             Defaults.get_system_mount_info_file()
         )
         Command.run(
-            ['mount', '--bind', sysconfig_network, '/etc/sysconfig/network']
+            [
+                'mount', '--bind', sysconfig_network_providers,
+                '/etc/sysconfig/network/providers'
+            ]
         )
         system_mount.add_entry(
-            sysconfig_network, '/etc/sysconfig/network'
+            sysconfig_network_providers, '/etc/sysconfig/network/providers'
         )
+        for interface_setup in glob.glob(sysconfig_interfaces):
+            if os.path.isfile(interface_setup):
```

Comment:
Small naming nitpick: `interface_setup` is not `interface_setup` if it is a directory.

Reply:
yeah, I'll find a better name

Reply:
how about it now ?

Reply:
Yeah, it's hard to come up with a good name for this. `network_... stuff`? :D But I guess `network_setup` works.